CHANGELOG.md

```diff
@@ -1,7 +1,30 @@
 # Release notes
 This page documents production updates to Cloud Code for IntelliJ. You can check this page for announcements about new or updated features, bug fixes, known issues, and deprecated functionality.
 
-## 20.2.1 - Latest Release
+## 20.4.1 - Latest Release
+
+We are pleased to announce [Cloud Run](https://cloud.google.com/run) support in Cloud Code for IntelliJ!
+
+Cloud Code now makes it easy to deploy your services to Cloud Run fully managed or Cloud Run for Anthos on GKE. You can view your deployed services right in the IDE with the Cloud Run Explorer.
+
+Follow the [Cloud Run quickstart guide](https://cloud.google.com/code/docs/intellij/quickstart-cloud-run), and use our built-in templates to get started quickly. Please file any feature requests or issues you encounter on our [GitHub repo](https://github.com/GoogleCloudPlatform/cloud-code-intellij/issues/new)!
+
+![view remote yaml](docs/images/release-notes/cloud-run.png)
```

Comment:
`view remote yaml` seems to be an error for an alternative text?

Reply:
good catch

Reply:
fixed